src/main/java/pl/damianszczepanik/jenkins/buildhistorymanager/descriptors/actions/DeleteArtifactsWithPatternActionDescriptor.java

```diff
@@ -0,0 +1,53 @@
+package pl.damianszczepanik.jenkins.buildhistorymanager.descriptors.actions;
+
+import java.util.logging.Logger;
+
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import hudson.Extension;
+import hudson.model.Descriptor;
+import pl.damianszczepanik.jenkins.buildhistorymanager.model.actions.Action;
+import pl.damianszczepanik.jenkins.buildhistorymanager.model.actions.DeleteArtifactsWithPatternAction;
+
+/**
+ * Descriptor implementation needed to render UI for {@link DeleteArtifactsWithPatternAction}.
+ */
+@Extension
+@Symbol("DeleteArtifactsWithPattern")
+public class DeleteArtifactsWithPatternActionDescriptor extends Descriptor<Action>{
+    private static final Logger LOGGER = Logger.getLogger(DeleteArtifactsWithPatternActionDescriptor.class.getName());
+
+    private String include;
+    private String exclude;
+
+    public String getInclude() {
+        return include;
+    }
```

Comment:
any plan for this? now or later...

Reply:
do you prefer to exclude this class from code coverage or to add tests for getters and setters?

Reply:
add test